### PR TITLE
ui: fix create k8s cluster networks listing

### LINKED ISSUE #8536: Networks are getting listed in the create kubernetes form
ISSUE TYPE

Bug

COMPONENT NAME

Component: UI

CLOUDSTACK VERSION

Cloudstack version 4.19 rc 2

SUMMARY

Networks are getting listed in the create kubernetes form

Steps to reproduce the issue 

1. Navigate to compute > Kubernetes > Create Kubernetes cluster 

The network selection is listing the network twice.

Cloudstack is calling the listnetworks api call twice

Screenshot

k8s419rc2

k8s419rc

---
**This issue was not observed in 4.18 release, where the listNetworks call was being called only once**

k8s4182

k8s418

### PR DESCRIPTION
**Pull request by shwstppr**
### Description

Fixes #8536

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### Screenshots (if appropriate):

https://github.com/apache/cloudstack/assets/153340/f9f26f0f-b8ed-44ff-8fda-6cf2f8da8b0d

### CODE REVIEW

**Comment by harikrishna-patnala** [comment]
@blueorangutan ui

**Review by harikrishna-patnala — APPROVED** [review]
code LGTM

File: ui/src/views/compute/CreateKubernetesCluster.vue [hunk 1]
```diff
@@ -322,7 +316,6 @@ export default {
     },
     fetchData () {
       this.fetchZoneData()
-      this.fetchNetworkData()
```
Comment (shwstppr) [inline ui/src/views/compute/CreateKubernetesCluster.vue:325]: listNetworks will be listed on zone change

**Comment by shwstppr** [comment]
@blueorangutan ui

**Comment by shwstppr** [comment]
@blueorangutan package

**Comment by shwstppr** [comment]
Will back port to 4.18 branch after 4.19.0 RC3

**Review by kiranchavala — APPROVED** [review]
LGTM, Tested manually

The networks are listed only once when create kubernetes  cluster form is called in the Ui

and Also the zone selection is honored 

Screenshot 2024-01-22 at 10 00 54 AM

**Pull request merged**